src/foremast/utils/vpc.py

```diff
@@ -51,7 +51,7 @@ def get_vpc_id(account, region):
 
     for vpc in vpcs:
         LOG.debug('VPC: %(name)s, %(account)s, %(region)s => %(id)s', vpc)
-        if all([vpc['name'] == 'vpc', vpc['account'] == account, vpc['region'] == region]):
+        if 'name' in vpc and all([vpc['name'] == 'vpc', vpc['account'] == account, vpc['region'] == region]):
```

Comment:
`name` wasn't in the new output?
Are you concerned about the `account` or `region` missing?

Reply:
Given how CloudDriver operates around Account and Region I think those are pretty hard set. The biggest issue is in foremast our vpc must be hardcoded named to vpc. I believe there is an issue with this.

After upgrading to the latest version of Spinnaker, I am seeing my default VPC in addition to my "vpc" where we do our operations. 

BTW, not a fan of my if check, if there is a more optimal let me know! 👍 Only did this as a quick fix. :)

Reply:
We already have an issue filed on the `vpc` naming issue https://github.com/foremast/foremast/issues/189.

Reply:
I agree, this will work for now.